lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -89,7 +89,7 @@
           percentage="main.state.getFlashState().percentage"
           striped="{{ main.state.getFlashState().type == 'check' }}"
           ng-attr-active="{{ main.state.isFlashing() }}"
-          ng-click="flash.flashImageToDrive(main.selection.getImagePath(), main.selection.getDrive())"
+          ng-click="flash.flashImageToDrive(main.selection.getImage(), main.selection.getDrive())"
```

Comment:
I guess its fine given that the reason for passing the whole image instead of just the path only makes sense because of analytics purposes.